pkg/controllers/clusterresourceplacement/placement_status.go

```diff
@@ -100,7 +100,7 @@ func buildClusterResourcePlacementApplyCondition(crp *fleetv1beta1.ClusterResour
 		}
 	}
 
-	if !isSync || pendingCount > 0 {
```

Comment:
This pendingCount means the work status itself is in the pending and isSync means whether the work has been updated to the latest resource. They're different.

Reply:
But with isSync <-> (pendCount == 0) there's no need to do the comparison here I am afraid.

Reply:
i.e., if isSync is false, syncPendingCount must be > 0, since isSync is determined by pendingCount == 0 and pendingCount cannot be negative.

Reply:
Discussed with Zhiying offline; it was a missight on my end, the variable is about applied parts.